### PR TITLE
Fix for installation error

### LINKED ISSUE #6: Reference error when installing the plugin
The following error is printed out when the plugins is installed:

```shell
$ twilio plugins:install kaiquelupo/plugin-twilio-infra

....

(node:19617) ReferenceError Plugin: plugin-twilio-infra: cliInfo is not defined
module: @oclif/config@1.17.0
task: toCached
plugin: plugin-twilio-infra
root: /Users/twilio/.twilio-cli/node_modules/plugin-twilio-infra
See more details with DEBUG=*
Installing plugin https://github.com/kaiquelupo/plugin-twilio-infra... installed v0.0.1
```

The issue is with this line https://github.com/kaiquelupo/plugin-twilio-infra/blob/b40a4f2e30de6144082cdf97b299c0e64e8ca776/src/commands/infra/stack/new.js#L42

### PR DESCRIPTION
**Pull request by vernig**
* Fix #6 
* Remove some reference to serverless plugin project

### CODE REVIEW

File: package.json
Comment (bermudezmt) [inline package.json:39]: Maybe using [$ref](https://swagger.io/docs/specification/using-ref/) is overkill for this JSON file, but let's remember to update this URL later.

File: src/commands/infra/stack/new.js
Comment (bermudezmt) [inline src/commands/infra/stack/new.js:42]: Hmm, I checked out your branch, uninstalled the twilio-infra plugin then reinstalled. I'm still seeing the error:
```
(node:66188) ReferenceError Plugin: plugin-twilio-infra: cliInfo is not defined
module: @oclif/config@1.17.0
task: toCached
plugin: plugin-twilio-infra
root: /Users/mbermudez/.twilio-cli/node_modules/plugin-twilio-infra
See more details with DEBUG=*
Installing plugin https://github.com/kaiquelupo/plugin-twilio-infra... installed v0.0.1
```

Is this the right installation command? `twilio plugins:install kaiquelupo/plugin-twilio-infra`

Comment (vernig) [reply src/commands/infra/stack/new.js:42]: @bermudezmt are you testing the changes using:

```shell
twilio plugins:link <path_to_your_local_repo>
````

From your log it looks like you are still fetching the repo from github.

File: package.json
Comment (vernig) [reply package.json:39]: Very good point @bermudezmt!

File: src/commands/infra/stack/new.js
Comment (bermudezmt) [reply src/commands/infra/stack/new.js:42]: Yeah you're right. It worked but after running `npm install` first. :) I created a PR  for updating the README. Can you please review? I can't tag you somehow.

**Review by bermudezmt — APPROVED** [review]
LGTM. Thanks, @vernig

**Comment by vernig** [comment]
@kaiquelupo can you approve as well? Once merged this and #8 we can merge everything to https://github.com/twilio-infra-as-code/plugin-twilio-infra

**Pull request merged**
